hathor/builder/cli_builder.py

```diff
@@ -270,6 +270,11 @@ def create_manager(self, reactor: Reactor) -> HathorManager:
             cpu_mining_service=cpu_mining_service
         )
 
+        if self._args.x_ipython_kernel:
+            self.check_or_raise(self._args.x_asyncio_reactor,
+                                '--x-ipython-kernel must not be used with --x-asyncio-reactor')
```

Comment:
I think the error message is inverted, isn't it?

Reply:
Oh, right!

Reply:
Fixed.

Reply:
Thanks!